src/tqec/plaquette/rpng/rpng.py

```diff
@@ -16,6 +17,22 @@ def __str__(self) -> str:
     def to_extended_basis(self) -> ExtendedBasis:
         return ExtendedBasis(self.value)
 
+    def to_dict(self) -> dict[str, Any]:
+        return {"value": self.value}
```

Comment:
Is that needed? Can't we just represent a `PauliBasis` by its value instead of having another indirection?